README.md

```diff
@@ -14,6 +14,7 @@ images to your devices.
 
 To learn more about openBalena, visit [balena.io/open][open-balena-website].
 
+> **IMPORTANT:** Due to changes in [balenaOS][balena-os], only versions up to and including `2.49.0` are currently supported.
```

Comment:
I also think it makes sense to move this down below on the "Getting started" section.